modell/src/main/kotlin/no/nav/tilbakekreving/Tilbakekreving.kt

```diff
@@ -11,11 +16,13 @@ import no.nav.tilbakekreving.tilstand.Tilstand
 import java.time.LocalDateTime
 
 class Tilbakekreving(
-    private val eksternFagsak: EksternFagsak,
+    val eksternFagsak: EksternFagsak,
+    val behandlingHistorikk: BehandlingHistorikk,
     private val opprettet: LocalDateTime,
+    private val behovOppservatør: BehovOppservatør,
```

Comment:
En rekke følgefeil her. Søker du gjennom prosjektet og fikser skrivefeilen?
```suggestion
    private val behovObservatør: BehovObservatør,
```